Makefile.cfg

```diff
@@ -12,7 +12,6 @@ HOSTLDFLAGS		= -L/usr/local/lib
 # For MinGW/MSYS, MinGW-w64/MSYS2 and Cygwin
 ifdef WINDOWS
   HOSTCFLAGS += -D_WIN32
-  HOSTLDFLAGS += -static
```

Comment:
This is undoing something in a previous PR. Remove.

Reply:
Removed